# Output from Extended Kalman Filter

Header header

# Drone position
float64 x
float64 y
float64 z

# Drone orientation
float64 phi
float64 theta
float64 psi
